resources/v2/01_students.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE students
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    name text NOT NULL,
+    city text NOT NULL,
+    age integer NOT NULL,
+    
+    created_at timestamp
+    WITH time zone DEFAULT NOW(),
+    subject_id integer NOT NULL
+    updated_at timestamp
+    WITH time zone,
+    deleted_at timestamp
+    WITH time zone
+);
+
+CREATE INDEX students_id ON students USING btree (id)
+CREATE INDEX students_name ON students USING btree (name)
+CREATE INDEX students_city ON students USING btree (city)
```

Comment:
You shouldnt index for everything and anything. It will slow down your writes. Only what is required
So. Remove these for now because you still dont know if you need index here or no

Reply:
Won't we require searching on students table via `name` and `city` ?

Reply:
Took the reference from the template only.

Reply:
You could. But are you gonna write some kind of query to filter by city and filter by name ?
Maybe city you might require but how about name ? When would you filter by name ?

Usually we would try to find a student by his id. Same goes with subject as well. Unless you think that there is a usecase in the query you are writing and you think that you need to speeded it, no need to index that column